provision/provision.sh

```diff
@@ -102,6 +102,16 @@ apt_package_install_list=(
 
 ### FUNCTIONS
 
+is_utility_installed() {
+  local utilities=`cat ${VVV_CONFIG} | shyaml get-values utilities.${1} 2> /dev/null`
```

Comment:
not sure about that command, this should be `utilities.core.${1}`?

Reply:
oh bugger I see what you mean

Reply:
Fixed in the release PR to `develop` #1713